src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -240,8 +245,10 @@ public Result Result(string query, string queryArguments, IPublicAPI api)
             result.TitleHighlightData = StringMatcher.FuzzySearch(query, Name).MatchData;
 
             // Using CurrentCulture since this is user facing
-            var toolTipTitle = string.Format(CultureInfo.CurrentCulture, "{0}: {1}", Properties.Resources.powertoys_run_plugin_program_file_name, result.Title);
-            var toolTipText = string.Format(CultureInfo.CurrentCulture, "{0}: {1}", Properties.Resources.powertoys_run_plugin_program_file_path, FullPath);
+            string toolTipLocalizedName = !string.IsNullOrEmpty(LocalizedName) ? LocalizedName : result.Title;
+            string toolTipNonLocalizedName = !string.IsNullOrEmpty(LocalizedName) && !(LocalizedName == Name) ? string.Format(CultureInfo.CurrentCulture, "{0}: {1}", Properties.Resources.powertoys_run_plugin_program_org_name, Name) + "\n" : string.Empty;
```

Comment:
Is this an information that is important for the user? 🤔 After thinking about it I feel that this can de removed.

Reply:
I will remove it as this information isn't useful for an user.